### PR TITLE
bump to laravel 8

### PR DESCRIPTION
**Pull request by adlerdias**
bump requirements to meet laravel 8 requirements.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "kabbouchi/laravel-logger-discord-channel",
+  "name": "adlerdias/laravel-logger-discord-channel",
```
Comment (KABBOUCHI) [inline composer.json]: can u revert this change?:)

**Review by KABBOUCHI — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (adlerdias) [reply composer.json]: Of course I can, I'm really sorry about that.
